roles/foreman_provisioning/tasks/main.yml

```diff
@@ -9,12 +14,6 @@
   set_fact:
     foreman_provisioning_smart_proxy: "{{ foreman_provisioning_smart_proxy_json.stdout|from_json }}"
 
-- name: 'install libvirt gem'
-  yum: name=rubygem-ruby-libvirt state=present
-
-- name: 'install foreman-libvirt'
```

Comment:
I think you need to keep this one as well, otherwise there will be no libvirt compute resource.

Reply:
I thought running the installer with ```--enable-foreman-compute-libvirt``` would take care of that.